### ISSUE TITLE
Spelling mistake in options.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6438]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6438|
|Type|defect|
|Created|2013-04-17 13:14:38Z|

`doc/core/howto/options.xhtml` says "rathers", which should be "rather".

Attachments:

* [options_xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/830/830f26a7c080db30e81312d2d6932c79cea81a92/ad3c217ee885b4967e0c650849895b35207a8e1f.patch) (634 bytes) - added by chitrankdixit on 2013-04-21 04:33:32Z - options.xhtml patch resolved the spelling mistake

<details><summary>Searchable metadata</summary>

```
trac-id__6438 6438
type__defect defect
reporter__itamar itamar
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1366204478000000 1366204478000000
changetime__1366916360000000 1366916360000000
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_1"></a>|@itamarst commented|
|-|-|

Also "hiearchy" is an error which should be fixed.

**Comment by twisted-trac** [comment]
|<img alt="Infinity's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Infinity set owner to Infinity<br>Infinity set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="Infinity's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Infinity removed owner<br>Infinity set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch, I'll apply it. Make sure to create your patch from the root of your checkout next time, as described [TwistedDevelopment#SubmittingaPatch here].

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38261]) Apply options_xhtml.patch: Correct a typo in the usage.Options howto.

Author: chitrankdixit
Reviewer: thijs
Fixes: [#6438](https://github.com/twisted/twisted/issues/6438)
